cypress/support/pageObjects/OrangeHRM/addEmployee.ts

```diff
@@ -63,27 +85,98 @@ class addEmployee {
     });
   }
 
-  createLoginDetails(username: string, password: string) {
-    const currentDate = new Date();
+  createLoginDetails(username: string, password: string): Cypress.Chainable<any> {
+    return cy.wrap(undefined).then(() => {
+      cy.request({
+        method: "POST",
+        url: "/web/index.php/api/v2/admin/users",
+        body: {
+          empNumber: this.getEmpNumber(),
+          password: password,
+          status: true,
+          userRoleId: 2,
+          username: username
+        },
+      }).then((response) => {
+        console.log(response.body.data.userName);
+        this.setUserName(response.body.data.userName);
+        expect(response).property("status").to.equal(200);
```

Comment:
No need for this assertion cuz we use the API here not make API testing to check if the API is success or not